frame/staking/README.md

```diff
@@ -50,8 +50,9 @@ used.
 
 An account pair can become bonded using the [`bond`](https://docs.rs/pallet-staking/latest/pallet_staking/enum.Call.html#variant.bond) call.
 
-Stash accounts can change their associated controller using the
-[`set_controller`](https://docs.rs/pallet-staking/latest/pallet_staking/enum.Call.html#variant.set_controller) call.
+Stash accounts can update their associated controller back to their stash account using the
+[`set_controller`](https://docs.rs/pallet-staking/latest/pallet_staking/enum.Call.html#variant.set_controller)
+call.
```

Comment:
nit since this is temporary, but probably useful to mention in the README that controllers in staking are being deprecated, thus it is not possible to set a controller for a stash anymore.